connectors/sources/tests/fixtures/fixture.py

```diff
@@ -100,7 +100,7 @@ def main(args=None):
         os.chdir(os.path.join(os.path.dirname(__file__), args.name))
         if args.action == "start_stack":
             os.system("docker compose up -d")
-            # XXX do better
+            # TODO: do better
```

Comment:
I want to change `do better` but I'm not even sure what it's referencing. I assume the sleep below it but I don't know.

Reply:
Makes sense, this will be addressed in https://github.com/elastic/enterprise-search-team/issues/3853.